nemo_curator/modules/dataset_ops.py

```diff
@@ -136,23 +136,23 @@ def blend_datasets(
             the final blend.
     """
     if len(datasets) != len(sampling_weights):
-        raise ValueError(
+        msg = (
             f"Different number of datasets and weights specified. {len(datasets)} datasets and {len(sampling_weights)}"
         )
+        raise ValueError(msg)
 
     weight_sum = sum(sampling_weights)
     sampling_weights = [weight / weight_sum for weight in sampling_weights]
-    num_documents_per_dataset = [
-        math.ceil(weight * target_size) for weight in sampling_weights
-    ]
+    num_documents_per_dataset = [math.ceil(weight * target_size) for weight in sampling_weights]
 
     blend_components = []
-    for dataset, num_documents in zip(datasets, num_documents_per_dataset):
+    for dataset, num_documents in zip(datasets, num_documents_per_dataset, strict=False):
```

Comment:
Shouldn't this be `strict=True`?

Reply:
By default Python 3.10 sets `strict=False`, so the above should maintain the same behavior as before.